lib/msal-browser/docs/v4-migration.md

```diff
@@ -314,6 +315,94 @@ const authRequest = {
 
 > Note: In cases where MSAL determines `extraParameters` must be encoded into the URL string, `extraParameters` will be merged with `extraQueryParams` in a way that will cause same-named parameters to be overwritten. In these cases, the value for the parameter in `extraParameters` will take precedence over the value in the `extraQueryParams`.
 
+### Cross-Origin-Opener-Policy (COOP) Support
+
+MSAL Browser v5 introduces built-in support for Cross-Origin-Opener-Policy (COOP), which enhances security by isolating your browsing context. When COOP is enabled on your application, popup and iframe-based authentication flows are restricted. MSAL v5 provides a redirect bridge mechanism to handle authentication in COOP-enabled environments.
```

Comment:
Isn't COOP going to be enabled server side? Does application policy matter anymore once enabled by the service?

Reply:
Fixed